tensorflow/lite/java/demo/app/build.gradle

```diff
@@ -1,20 +1,28 @@
 apply plugin: 'com.android.application'
 
+buildscript {
+    repositories {
+        // Gradle 4.1 and higher include support for Google's Maven repo using
+        // the google() method. And you need to include this repo to download
+        // Android Gradle plugin 3.0.0 or higher.
+        google()
+        mavenCentral()
```

Comment:
Can you explain why we need mavenCentral()?

Reply:
Otherwise lgtm.

Reply:
If I remove both the `mavenCentral()` lines and do a `Sync Project with Gradle Files`, then I hit this error:
```
Could not find org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71.
Searched in the following locations:
    https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.2.71/kotlin-stdlib-jdk8-1.2.71.pom
    https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.2.71/kotlin-stdlib-jdk8-1.2.71.jar
Required by:
    project :app > com.android.tools.build:gradle:3.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.build:builder:3.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.lint:lint-gradle-api:26.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.build:gradle-api:3.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > androidx.databinding:databinding-compiler-common:3.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.build:builder:3.2.1 > com.android.tools:sdk-common:26.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.build:builder:3.2.1 > com.android.tools:common:26.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.build:builder:3.2.1 > com.android.tools.build:manifest-merger:26.2.1
    project :app > com.android.tools.build:gradle:3.2.1 > com.android.tools.build:builder:3.2.1 > com.android.tools:sdklib:26.2.1 > com.android.tools:repository:26.2.1
```

Reply:
Running `./gradlew app:dependencies` shows that the Kotlin dependency comes from the `lintClassPath`:
```
lintClassPath - The lint embedded classpath
\--- com.android.tools.lint:lint-gradle:26.2.1
     +--- com.android.tools:sdk-common:26.2.1
     |    +--- com.android.tools:sdklib:26.2.1
     |    |    +--- com.android.tools.layoutlib:layoutlib-api:26.2.1
     |    |    |    +--- com.android.tools:common:26.2.1
     |    |    |    |    +--- com.android.tools:annotations:26.2.1
     |    |    |    |    +--- com.google.guava:guava:23.0
     |    |    |    |    |    +--- com.google.code.findbugs:jsr305:1.3.9
     |    |    |    |    |    +--- com.google.errorprone:error_prone_annotations:2.0.18
     |    |    |    |    |    +--- com.google.j2objc:j2objc-annotations:1.1
     |    |    |    |    |    \--- org.codehaus.mojo:animal-sniffer-annotations:1.14
     |    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71
     |    |    |    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.2.71
     |    |    |    |         |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.2.71
     |    |    |    |         |    \--- org.jetbrains:annotations:13.0
     |    |    |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.71
     |    |    |    |              \--- org.jetbrains.kotlin:kotlin-stdlib:1.2.71 (*)
     |    |    |    +--- net.sf.kxml:kxml2:2.3.0
     |    |    |    +--- com.android.tools:annotations:26.2.1
     |    |    |    \--- org.jetbrains:annotations:13.0
     |    |    +--- com.android.tools:dvlib:26.2.1
     |    |    |    \--- com.android.tools:common:26.2.1 (*)
     |    |    +--- com.android.tools:repository:26.2.1
     |    |    |    +--- com.android.tools:common:26.2.1 (*)
     |    |    |    +--- com.sun.activation:javax.activation:1.2.0
     |    |    |    +--- org.apache.commons:commons-compress:1.12
     |    |    |    +--- org.glassfish.jaxb:jaxb-runtime:2.2.11
     |    |    |    |    +--- org.glassfish.jaxb:jaxb-core:2.2.11
     |    |    |    |    |    +--- javax.xml.bind:jaxb-api:2.2.12-b140109.1041
     |    |    |    |    |    +--- org.glassfish.jaxb:txw2:2.2.11
     |    |    |    |    |    \--- com.sun.istack:istack-commons-runtime:2.21
     |    |    |    |    +--- org.jvnet.staxex:stax-ex:1.7.7
     |    |    |    |    \--- com.sun.xml.fastinfoset:FastInfoset:1.2.13
     |    |    |    +--- com.google.jimfs:jimfs:1.1
     |    |    |    |    \--- com.google.guava:guava:18.0 -> 23.0 (*)
     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    |    +--- com.google.code.gson:gson:2.8.0
     |    |    +--- org.apache.commons:commons-compress:1.12
     |    |    +--- org.apache.httpcomponents:httpmime:4.5.2
     |    |    |    \--- org.apache.httpcomponents:httpclient:4.5.2
     |    |    |         +--- org.apache.httpcomponents:httpcore:4.4.4 -> 4.4.5
     |    |    |         +--- commons-logging:commons-logging:1.2
     |    |    |         \--- commons-codec:commons-codec:1.9
     |    |    \--- org.apache.httpcomponents:httpcore:4.4.5
     |    +--- com.android.tools.build:builder-test-api:3.2.1
     |    |    \--- com.android.tools.ddms:ddmlib:26.2.1
     |    |         +--- com.android.tools:common:26.2.1 (*)
     |    |         \--- net.sf.kxml:kxml2:2.3.0
     |    +--- com.android.tools.build:builder-model:3.2.1
     |    |    \--- com.android.tools:annotations:26.2.1
     |    +--- com.android.tools.ddms:ddmlib:26.2.1 (*)
     |    +--- org.bouncycastle:bcpkix-jdk15on:1.56
     |    |    \--- org.bouncycastle:bcprov-jdk15on:1.56
     |    +--- org.bouncycastle:bcprov-jdk15on:1.56
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    +--- org.jetbrains.kotlin:kotlin-reflect:1.2.0
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.2.0 -> 1.2.71 (*)
     |    +--- com.google.protobuf:protobuf-java:3.4.0
     |    \--- javax.inject:javax.inject:1
     +--- com.android.tools.build:builder:3.2.1
     |    +--- com.android.tools.build:builder-model:3.2.1 (*)
     |    +--- com.android.tools.build:builder-test-api:3.2.1 (*)
     |    +--- com.android.tools:sdklib:26.2.1 (*)
     |    +--- com.android.tools:sdk-common:26.2.1 (*)
     |    +--- com.android.tools:common:26.2.1 (*)
     |    +--- com.android.tools.build:manifest-merger:26.2.1
     |    |    +--- com.android.tools:common:26.2.1 (*)
     |    |    +--- com.android.tools:sdklib:26.2.1 (*)
     |    |    +--- com.android.tools:sdk-common:26.2.1 (*)
     |    |    +--- com.google.code.gson:gson:2.8.0
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    |    \--- net.sf.kxml:kxml2:2.3.0
     |    +--- com.android.tools.ddms:ddmlib:26.2.1 (*)
     |    +--- com.android.tools.analytics-library:protos:26.2.1
     |    |    \--- com.google.protobuf:protobuf-java:3.4.0
     |    +--- com.android.tools.analytics-library:shared:26.2.1
     |    |    +--- com.android.tools.analytics-library:protos:26.2.1 (*)
     |    |    +--- com.android.tools:annotations:26.2.1
     |    |    +--- com.android.tools:common:26.2.1 (*)
     |    |    +--- com.google.guava:guava:23.0 (*)
     |    |    \--- com.google.code.gson:gson:2.8.0
     |    +--- com.android.tools.analytics-library:tracker:26.2.1
     |    |    +--- com.android.tools:annotations:26.2.1
     |    |    +--- com.android.tools:common:26.2.1 (*)
     |    |    +--- com.android.tools.analytics-library:protos:26.2.1 (*)
     |    |    +--- com.android.tools.analytics-library:shared:26.2.1 (*)
     |    |    +--- com.google.protobuf:protobuf-java:3.4.0
     |    |    \--- com.google.guava:guava:23.0 (*)
     |    +--- com.android.tools.build:apksig:3.2.1
     |    +--- com.android.tools.build:apkzlib:3.2.1
     |    |    +--- com.google.code.findbugs:jsr305:1.3.9
     |    |    +--- com.google.guava:guava:23.0 (*)
     |    |    +--- org.bouncycastle:bcpkix-jdk15on:1.56 (*)
     |    |    +--- org.bouncycastle:bcprov-jdk15on:1.56
     |    |    \--- com.android.tools.build:apksig:3.2.1
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    +--- com.squareup:javawriter:2.5.0
     |    +--- org.bouncycastle:bcpkix-jdk15on:1.56 (*)
     |    +--- org.bouncycastle:bcprov-jdk15on:1.56
     |    +--- org.ow2.asm:asm:6.0
     |    +--- org.ow2.asm:asm-tree:6.0
     |    |    \--- org.ow2.asm:asm:6.0
     |    +--- org.ow2.asm:asm-commons:6.0
     |    |    \--- org.ow2.asm:asm-tree:6.0 (*)
     |    +--- org.ow2.asm:asm-util:6.0
     |    |    \--- org.ow2.asm:asm-tree:6.0 (*)
     |    +--- it.unimi.dsi:fastutil:7.2.0
     |    +--- net.sf.jopt-simple:jopt-simple:4.9
     |    \--- com.googlecode.json-simple:json-simple:1.1
     +--- com.android.tools.build:builder-model:3.2.1 (*)
     +--- com.android.tools.external.com-intellij:intellij-core:26.2.1
     |    \--- org.jetbrains.trove4j:trove4j:20160824
     +--- com.android.tools.external.org-jetbrains:uast:26.2.1
     +--- com.android.tools.build:manifest-merger:26.2.1 (*)
     +--- com.android.tools.lint:lint:26.2.1
     |    +--- com.android.tools.lint:lint-checks:26.2.1
     |    |    +--- com.android.tools.lint:lint-api:26.2.1
     |    |    |    +--- com.android.tools:sdk-common:26.2.1 (*)
     |    |    |    +--- com.android.tools.build:builder-model:3.2.1 (*)
     |    |    |    +--- com.google.guava:guava:23.0 (*)
     |    |    |    +--- com.android.tools.lint:lint-kotlin:26.2.1
     |    |    |    |    +--- com.android.tools.external.com-intellij:intellij-core:26.2.1 (*)
     |    |    |    |    +--- com.android.tools.external.com-intellij:kotlin-compiler:26.2.1
     |    |    |    |    +--- com.android.tools.external.org-jetbrains:uast:26.2.1
     |    |    |    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.2.0 (*)
     |    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    |    |    +--- com.android.tools.external.com-intellij:intellij-core:26.2.1 (*)
     |    |    |    +--- com.android.tools.external.org-jetbrains:uast:26.2.1
     |    |    |    +--- com.android.tools.build:manifest-merger:26.2.1 (*)
     |    |    |    +--- org.ow2.asm:asm:6.0
     |    |    |    +--- org.ow2.asm:asm-tree:6.0 (*)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.2.0 (*)
     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    |    +--- com.google.guava:guava:23.0 (*)
     |    |    +--- com.android.tools.external.com-intellij:intellij-core:26.2.1 (*)
     |    |    +--- com.android.tools.external.org-jetbrains:uast:26.2.1
     |    |    \--- org.ow2.asm:asm-analysis:6.0
     |    |         \--- org.ow2.asm:asm-tree:6.0 (*)
     |    +--- com.android.tools.lint:lint-kotlin:26.2.1 (*)
     |    +--- com.google.guava:guava:23.0 (*)
     |    +--- com.android.tools.external.org-jetbrains:uast:26.2.1
     |    +--- com.android.tools.external.com-intellij:kotlin-compiler:26.2.1
     |    +--- com.android.tools.build:manifest-merger:26.2.1 (*)
     |    +--- com.android.tools.analytics-library:shared:26.2.1 (*)
     |    +--- com.android.tools.analytics-library:protos:26.2.1 (*)
     |    +--- com.android.tools.analytics-library:tracker:26.2.1 (*)
     |    +--- org.jetbrains.kotlin:kotlin-reflect:1.2.0 (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     +--- com.android.tools.lint:lint-gradle-api:26.2.1
     |    +--- com.android.tools:sdklib:26.2.1 (*)
     |    +--- com.android.tools.build:builder-model:3.2.1 (*)
     |    +--- com.android.tools.build:gradle-api:3.2.1
     |    |    +--- com.android.tools.build:builder-model:3.2.1 (*)
     |    |    +--- com.android.tools.build:builder-test-api:3.2.1 (*)
     |    |    +--- com.google.guava:guava:23.0 (*)
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    +--- org.jetbrains.kotlin:kotlin-reflect:1.2.0 (*)
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
     |    \--- com.google.guava:guava:23.0 (*)
     +--- org.codehaus.groovy:groovy-all:2.4.12
     +--- org.jetbrains.kotlin:kotlin-reflect:1.2.0 (*)
     \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.71 (*)
```